renderer/index.css

```diff
@@ -827,6 +827,11 @@ body.drag .app::after {
   outline: none;
 }
 
+.player-controls .time {
```

Comment:
I wouldn't worry about it, for now. I plan to send a PR soon to cleanup the player controls.